features/src/main/feature/camel-features.xml

```diff
@@ -2157,21 +2157,19 @@
 <!--        <bundle dependency='true'>mvn:org.apache.thrift/libthrift/${zipkin-libthrift-version}</bundle>-->
 <!--        <bundle>mvn:org.apache.camel/camel-zipkin/${project.version}</bundle>-->
 <!--    </feature>-->
-<!--    <feature name='camel-zookeeper' version='${project.version}' start-level='50'>-->
-<!--        <feature version='${project.version}'>camel-core</feature>-->
-<!--        <bundle dependency='true'>wrap:mvn:org.apache.zookeeper/zookeeper/${zookeeper-version}$Export-Package=org.apache.zookeeper;version=${zookeeper-version},org.apache.zookeeper.*;version=${zookeeper-version}</bundle>-->
-<!--        <bundle dependency='true'>wrap:mvn:org.apache.zookeeper/zookeeper-jute/${zookeeper-version}$Export-Package=org.apache.jute;version=${zookeeper-version},org.apache.zookeeper.*;version=${zookeeper-version}</bundle>-->
-<!--        <bundle dependency='true'>mvn:org.apache.curator/curator-framework/${curator-version}</bundle>-->
-<!--        <bundle dependency='true'>mvn:org.apache.curator/curator-client/${curator-version}</bundle>-->
-<!--        <bundle dependency='true'>mvn:org.apache.curator/curator-recipes/${curator-version}</bundle>-->
-<!--        <bundle dependency='true'>mvn:org.apache.curator/curator-x-discovery/${curator-version}</bundle>-->
-<!--        <bundle dependency='true'>mvn:com.google.guava/failureaccess/1.0.1</bundle>-->
-<!--        <bundle dependency='true'>mvn:com.google.guava/guava/${zookeeper-guava-version}</bundle>-->
-<!--        <bundle dependency='true'>mvn:com.fasterxml.jackson.core/jackson-core/${jackson2-version}</bundle>-->
-<!--        <bundle dependency='true'>mvn:com.fasterxml.jackson.core/jackson-databind/${jackson2-version}</bundle>-->
-<!--        <bundle dependency='true'>mvn:com.fasterxml.jackson.core/jackson-annotations/${jackson2-version}</bundle>-->
-<!--        <bundle>mvn:org.apache.camel/camel-zookeeper/${project.version}</bundle>-->
-<!--    </feature>-->
+    <feature name='camel-zookeeper' version='${project.version}' start-level='50'>
+        <feature version='${project.version}'>camel-core</feature>
+        <feature>jackson</feature>
+        <bundle dependency='true'>wrap:mvn:org.apache.zookeeper/zookeeper/${zookeeper-version}$Export-Package=org.apache.zookeeper;version=${zookeeper-version},org.apache.zookeeper.*;version=${zookeeper-version}</bundle>
+        <bundle dependency='true'>wrap:mvn:org.apache.zookeeper/zookeeper-jute/${zookeeper-version}$Export-Package=org.apache.jute;version=${zookeeper-version},org.apache.zookeeper.*;version=${zookeeper-version}</bundle>
```

Comment:
Can be simplified using the pattern `org.apache.*`

Reply:
Ok done